src/main/java/se/bjurr/violations/lib/example/OtherClass.java

```diff
@@ -4,11 +4,14 @@
  * No ending dot
  */
 public class OtherClass {
- public static String CoNstANT = "yes";
+ public static String CoNstANT = "yes"; 
```

Comment:
**Reporter**: FINDBUGS
**Rule**: MS_SHOULD_BE_FINAL
**Severity**: INFO
**File**: se/bjurr/violations/lib/example/OtherClass.java L7
**Source**: se.bjurr.violations.lib.example.OtherClass

Field isn&#39;t final but should be&#10;&#10;&#10;   &lt;p&gt;&#10;This static field public but not final, and&#10;could be changed by malicious code or&#10;        by accident from another package.&#10;        The field could be made final to avoid&#10;        this vulnerability.&lt;/p&gt;&#10;


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1939212604>*